e107_languages/English/admin/lan_admin.php

```diff
@@ -201,6 +201,12 @@
 define("ADLAN_187", "Unable to create [x]. Please check your folder permissions.");
 define("ADLAN_188", "Your timezone setting [x] is invalid. It has been reset to UTC. To Modify, please go to Admin -> Preferences -> Date Display Options.");
 define("ADLAN_189", "The following plugins are not compatible with this version of e107 and should be uninstalled: ");
+define("ADLAN_190", "Need Help?");
+define("ADLAN_191", "Connect with our community for [x] with any upgrading issues you may encounter.");
+define("ADLAN_192", "Connect with our community for [x] with any e107 issues you may encounter.");
+define("ADLAN_193", "free support");
+define("ADLAN_194", "The folder [x] is not writable. Please correct before proceeding.");
+define("ADLAN_195", "Legacy avatars folder detected. Please make sure [x]avatars/ is writable. Please correct before proceeding.");
 
```

Comment:
There's a better way to add the link. ie. remove ADLAN_193. 
"Connect with our community for [free support] with any upgrading issues you may encounter". 

Then use `str_replace()` on `[` and `]` to place the `<a>` and `</a>` around 'free support'. etc.